pkg/util/net.go

```diff
@@ -81,21 +81,32 @@ func SubnetBroadcast(subnet string) string {
 		length = 128
 	}
 	maskLength, _ := cidr.Mask.Size()
+	if maskLength == 31 || maskLength == 127 {
+		return ""
+	}
```

Comment:
What if the `cidr` is an ipv6 one with prefix 31?

Reply:
Sorry, wanted to add also ipv6 and did mistake